toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -2,11 +2,36 @@
 
```

Comment:
I think we should add a Microsoft copyright notice, since this is a public repository.

Reply:
Added